cmd/okd/run.go

```diff
@@ -64,6 +65,7 @@ func NewRunCommand() *cobra.Command {
 	run.Flags().UintVar(&okdRunOpts.sshWorkerPort, "ssh-worker-port", 0, "port on localhost to ssh to worker node")
 	run.Flags().BoolVar(&okdRunOpts.background, "background", false, "go to background after nodes are up")
 	run.Flags().BoolVar(&okdRunOpts.randomPorts, "random-ports", true, "expose all ports on random localhost ports")
+	run.Flags().StringVar(&okdRunOpts.volume, "volume", "", "Bind mount a volume into the container")
```

Comment:
Will be the volume automatically unbound? Do we need some cleanup?

Reply:
This is still relevant

Reply:
According to this: https://github.com/fromanirh/pack8s/blob/master/cmd/rm.go#L87
all builtin volumes should be removed with container (https://github.com/containers/libpod/blob/e7540d0406c49b22de245246d16ebc6e1778df37/cmd/podman/varlink/io.podman.varlink#L733)

Reply:
This is fine. But does this also mean that the more `exec` we do on a container, the more volumes we pile in? what does it happen if we do two `pack8s exec` one after the other? will it work? will it pile up temporary volumes in the container?
Each call of `pack8s exec` must be idenpotent and do proper cleanup after itself; is not OK to rely on external tools (e.g. kubevirtci scripts) to do cleaning, because it's fragile and leads to surprising (and most likely unwanted) results.

Reply:
The volume is bind during start of container and persist till stop of container. It will not add or remove any volume during exec. I agree pack8s should handle cleanup and as i mentioned in previous comment, it removes all binded volumes with rm command

Reply:
I'm not 100% sure it's a good idea to have this volume bound between host and container, but since this is strictly for development and we are in the early stage of development, I'm incline to try it out and see how it looks.